recipes/ginkgo/all/conanfile.py

```diff
@@ -57,6 +57,9 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.16 <4]")
```

Comment:
only since 1.7.0, previous versions must not undergo specificities of other versions